rotkehlchen/types.py

```diff
@@ -499,3 +499,8 @@ class EnsMapping(NamedTuple):
     address: ChecksumEthAddress
     name: str
     last_update: Timestamp = Timestamp(0)
+
+
+class PnlOrder(DBEnumMixIn):
```

Comment:
Why DBEnumMixin? You don't need to store this in the DB. Just use a normal `SerializableEnumMixin`.

Also this should not be called `PnlOrder`. This is the accounting cost basis methodology used. So just call it `CostBasisMethod`

Reply:
Ok